src/SDKs/LocationBasedServices/Management.LocationBasedServices/Microsoft.Azure.Management.LocationBasedServices.csproj

```diff
@@ -1,13 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.LocationBasedServices</PackageId>
+    <PackageReleaseNotes>This package is deprecated. Please use https://www.nuget.org/packages/Microsoft.Azure.Management.Maps/ </PackageReleaseNotes>
```

Comment:
Let's make this a little more detailed
You can refer to an example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Microsoft.Azure.CognitiveServices.Language.csproj#L13)